schema.sql

```diff
@@ -6,4 +6,42 @@ CREATE TABLE animals (
     escape_attempt INT,
     neutered BOOLEAN,
     weight_kg DECIMAL
-);
\ No newline at end of file
+);
+
+CREATE TABLE owners(
+  id BIGSERIAL PRIMARY KEY NOT NULL,
+  full_name varchar(100) NOT NULL,
+  age int NOT NULL
+);
+
+CREATE TABLE species(
+  id bigsERIAL PRIMARY KEY NOT NULL,
+  name varchar(100) NOT NULL
+);
+
+ALTER TABLE animals ADD species varchar(100);
+
+ALTER TABLE animals DROP COLUMN species;
+
+ALTER TABLE animals ADD species_id BIGINt REFERENCES species(id);
+
+ALTER TABLE animals ADD owners_id BIGINT REFERENCES owners(id);
+
```

Comment:
### **Optional**
- [ ] Please remove all unnecessary code from the `schema.sql` file

Reply:
okay